src/VirtualClient/VirtualClient.Actions.UnitTests/SPEC/SpecJbbMetricsParserTests.cs

```diff
@@ -40,5 +40,22 @@ public void SpecJbbParserVerifyMetricsFpRate()
             MetricAssert.Exists(metrics, "max-jOPS", 4188, "jOPS");
             MetricAssert.Exists(metrics, "critical-jOPS", 1666, "jOPS");
         }
+
+
+        [Test]
+        public void SpecJbbParserVerifyNanMetricsFpRate()
```

Comment:
Please correct the name with fprate. If you copied it from elsewhere fix that too